src/components/report/TextControls.vue

```diff
@@ -43,51 +43,69 @@
 				</select>
 			</label>
 		</div>
-		<div>
-			Alignment
-			<span class="alignment-buttons">
-				<button
-					v-html="`&nbsp;`"
-					type="button"
-					class="button left"
-					:class="{ active: textAlign === 'left' }"
-					@click="align('left')"
-					title="Left align text"
-				></button>
-				<button
-					v-html="`&nbsp;`"
-					type="button"
-					class="button center"
-					:class="{ active: textAlign === 'center' }"
-					@click="align('center')"
-					title="Center align text"
-				></button>
-				<button
-					v-html="`&nbsp;`"
-					type="button"
-					class="button right"
-					:class="{ active: textAlign === 'right' }"
-					@click="align('right')"
-					title="Right align text"
-				></button>
-				<button
-					v-html="`&nbsp;`"
-					type="button"
-					class="button justify"
-					:class="{ active: textAlign === 'justify' }"
-					@click="align('justify')"
-					title="Justify text"
-				></button>
-				<button
-					type="button"
-					class="button initial"
-					:class="{ active: textAlign === 'initial' }"
-					@click="align('initial')"
-					title="Default alignment"
-				>
-					Default
-				</button>
-			</span>
+		<div class="alignment-buttons">
+			<span>Alignment</span>
+			<button
+				type="button"
+				class="button left"
+				:class="{ active: textAlign === 'left' }"
+				@click="align('left')"
+				title="Left align text"
+			>
+				<svg viewBox="0 0 18 18">
```

Comment:
Please add `focusable="false" aria-hidden="true"` to these SVGs for accessibility.

I leave https://www.scottohara.me/blog/2019/05/22/contextual-images-svgs-and-a11y.html#testing-the-svg-element for further reading.

Reply:
Good call, will add `aria-hidden`. The `focusable` is an IE thing relying on the outdated SVG Tiny spec, modern browsers will make `aria-hidden` unfocuasable.